test/test_autotag.py

```diff
@@ -749,13 +752,75 @@ def test_albumtype_applied(self):
         self.assertEqual(self.items[1].albumtype, 'album')
 
     def test_album_artist_overrides_empty_track_artist(self):
-        my_info = copy.deepcopy(self.info)
```

Comment:
One way to make a copy of a TrackInfo or AlbumInfo object would be to use `dict`'s [`update` method](https://docs.python.org/3/library/stdtypes.html#dict.update). Something like this:

```
trackinfo1 = TrackInfo(title='foo')
trackinfo2 = TrackInfo()
trackinfo2.update(trackinfo1)
```

That has the effect of copying all the keys & values from one dict into another.

Reply:
or even simpler: 
```
trackinfo1 = TrackInfo(title='foo')
trackinfo2 = TrackInfo(trackinfo1)
```
However, for updating ```AlbumInfo``` I need to recreate the tracks, don't I?

Reply:
Indeed; good point if that's still supported in `__init__`. Maybe a good way to do this would be to create a little function for duplicating `AlbumInfo` objects—something like this?

```py
def dup_albuminfo(ai):
    return AlbumInfo(ai, tracks=[TrackInfo(ti) for ti in ai.tracks])
```